examples/AlchemicalLoopSoftening.py

```diff
@@ -11,6 +11,25 @@
 from sams.tests.testsystems import SAMSTestSystem
 
 
+def minimize(testsystem, positions):
+    print("Minimizing...")
+    integrator = openmm.VerletIntegrator(1.0 * unit.femtoseconds)
```

Comment:
Change this to a Langevin integrator:
```python
integrator = openmm.LangevinIntegrator(300.0 * unit.kelvin, 90.0 / unit.picoseconds, 1.0 * unit.femtoseconds)
```
so that we can add some dynamics